src/details/ArborX_MinimumSpanningTree.hpp

```diff
@@ -0,0 +1,504 @@
+/****************************************************************************
+ * Copyright (c) 2017-2021 by the ArborX authors                            *
```

Comment:
```suggestion
 * Copyright (c) 2017-2022 by the ArborX authors                            *
```
here and other files.